### DISCUSSION TITLE
Choose what videos to download before it starts conversion

### DISCUSSION THREAD

**Discussion by EduSmasher** [shaked6540/YoutubePlaylistDownloader#289] [Ideas]
it's been working perfectly, one thing id add as a cherry on the top would be to be able to have a list of videos in whatever playlist youre downloading to drop down when you paste a link, and then you should be able to check/uncheck what videos you want to download before starting the conversion. i think ive read recently that you dont have much time to add new features but this would be a real favor to make this app perfect so at least id want you to know about it :)

**Comment by shaked6540** [comment]
This has been suggested a couple of times, I would love to add that when I have time. Right now though even if I have time, it won't be top priority since there are a couple of bugs that I need to fix first (mainly, bulk download not working, error messages not always showing etc)
